bin/netjsonconfig

```diff
@@ -146,11 +144,9 @@ def print_output(output):
     # if file object, get bytes
     if hasattr(output, 'getvalue'):
         output = output.getvalue()
-    # python2: always print
     # python3: if string, just print
-    if isinstance(output, six.string_types):
+    if isinstance(output, str):
         print(output)
-    # python2: never enters this block
     # python3: writes binary data to standard output
```

Comment:
Comment not required anymore.

Reply:
Hm! the comment is not present in my local change why it's present in here ?

Reply:
I am talking about "# python3: writes binary data to standard output".